### ISSUE TITLE
I'm so confused by redis sentinel

### ISSUE THREAD

**Issue by jeremych1000** [OT-CONTAINER-KIT/redis-operator#1165]
**Describe your question**
Hello! I'm trying to set up redis sentinel on Kubernetes, and have been using your operator for a while (just normal redis).

I'm very confused to how this operator sets up sentinel. 

Can I please ask some questions:
1. I understand sentinel watches redis. Does there have to be a 1:1 mapping between sentinel and redis, or can it be one to many?
2. The redis-sentinel helm chart seems to start a statefulset of redis-sentinel, however is this redis itself? Or do I need a statefulset of redis-sentinel, and one deployment of redis-replication?
3. I have a requirement to have a loadbalancer attached to each of my redis nodes, so that I can hit them directly (not through an ingress). How should I configure this? It seems the redis-sentinel chart only makes one service? I don't want it to be loadbalanced.

**What version of redis-operator are you using?**

redis-operator version: v0.18.1

**Comment by drivebyer** [comment]
> Does there have to be a 1:1 mapping between sentinel and redis, or can it be one to many?
Yes, a one-to-many (1:N) configuration is supported, as defined here: [GitHub Reference](https://github.com/OT-CONTAINER-KIT/redis-operator/blob/master/api/common_types.go#L142).

> The redis-sentinel helm chart seems to start a statefulset of redis-sentinel, however is this redis itself?
No, the StatefulSet consists only of Sentinel instances.

> I have a requirement to have a loadbalancer attached to each of my redis nodes, so that I can hit them directly (not through an ingress). How should I configure this?
Currently, this is not supported.

**Comment by shubham-cmyk** [comment]
You have to pair the redis-sentinel with the redis-replication since the redis-cluster has inbuilt failover
